readers/api/transport.go

```diff
@@ -30,7 +30,7 @@ var (
 	errInvalidRequest     = errors.New("received invalid request")
 	errUnauthorizedAccess = errors.New("missing or invalid credentials provided")
 	auth                  mainflux.ThingsServiceClient
-	queryFields           = []string{"subtopic", "publisher", "protocol", "name", "value", "v", "vs", "vb", "vd"}
+	queryFields           = []string{"subtopic", "publisher", "protocol", "name", "v", "vs", "vb", "vd", "from", "to"}
```

Comment:
Consider extracting these fields to a separate struct and sending them as a JSON.

Reply:
As explained here https://github.com/mainflux/mainflux/issues/1061 I'm considering to do it but I would to it in another PR.

Reply:
Is it a big work that demands an additional PR? If not - it is probably better to do it in this one.

Reply:
@drasko It have to be done for all readers. Similar to what we recently did for Things and Channels `PageMetadata`. IMO it's better to separate this in another PR.

Reply:
OK then, I approved. Please do not forget to do this in a subsequent PR. Joyeux Noël.